pkg/delete/delete.go

```diff
@@ -46,22 +49,35 @@ func (d DeletePointer) asCoordinate() coordinate.Coordinate {
 	}
 }
 
+type ClientSet struct {
+	DTClient         dtclient.Client
+	AutomationClient *automationClient.Client
+}
+
 // Configs removes all given entriesToDelete from the Dynatrace environment the given client connects to
-func Configs(client dtclient.Client, apis api.APIs, entriesToDelete map[string][]DeletePointer) []error {
+func Configs(clients ClientSet, apis api.APIs, automationResources map[string]config.AutomationResource, entriesToDelete map[string][]DeletePointer) []error {
```

Comment:
I'd strongly recommend to change the name of that function :D

Reply:
Agreed - forgot that in my refactoring notes. This one and the "per type" DeleteAll should IMO change